python/ray/tests/test_autoscaler_yaml.py

```diff
@@ -301,7 +301,7 @@ def testValidateLocal(self):
         ] = 0
         assert prepared_config == expected_prepared
 
-    def testValidateNetworkConfig(self):
+    def testValidateNetworkConfigForBackwardsCompatibility(self):
```

Comment:
Note that this test *will* break on this PR because we're intentionally breaking backwards compatibility.

Reply:
i have no idea what this is, maybe add a docstring